main.cpp

```diff
@@ -319,8 +324,22 @@ int main(int argc, char* argv[]) {
             server.prePoll(fdm);
             const uint64_t now = STimeNow();
             S_poll(fdm, max(nextActivity, now) - now);
-            nextActivity = STimeNow() + STIME_US_PER_S; // 1s max period
+            nextActivity = STimeNow() + 1'000'000; // 1s max period
             server.postPoll(fdm, nextActivity);
+
+            uint64_t end = STimeNow();
+            postPollCounter += (end - (nextActivity - 1'000'000));
```

Comment:
I'm having a real hard time figuring out what this logic means.  Can you add a comment explaining this?  I was kind fo expecting something obvious like:

```
uint64_t start = STimeNow(); // rename from `now`
server.postPoll(fdm, nextActivity);
uint64_end = STimeNow();
uint64_t elapsed = STimeNow() - end;
pollSum += elapsed;
pollCount++;
if (end > lastResetCounter + 10'000'000) {
    SINFO(...);
    pollSum = 0;
    pollCount = 0;
    lastResetCounter = end;
}
```